zerver/tests/test_openapi.py

```diff
@@ -149,6 +150,7 @@ def test_validate_against_openapi_schema(self) -> None:
                                         TEST_ENDPOINT,
                                         TEST_METHOD,
                                         TEST_RESPONSE_SUCCESS)
+        openapi.EXCLUDE_PROPERTIES = exclude_properties
```

Comment:
This sort of thing is generally better done with a try/finally or a context manage (`with mock.patch`).  Right now, if you run `test-backend --nonfatal-errors`, we could end up with later tests being broken once this block fails.

Reply:
Used try ... finally. Done